### ISSUE TITLE
MKOB - Provide 'application launcher'

### ISSUE THREAD

**Issue by AESilky** [MorseKOB/PyKOB#44]
There should be a platform friendly/specific 'application launcher' that can be installed and allows a user to launch the application via an icon/shortcut (rather than running `pythonX MKOB.py`) In a platform GUI. The app/launch icon should be a "Morse-type image". The icon should be shown in the 'start/active' bar and as a desktop item (if the user selects to create it) to launch the app or select it when it is running.

**Comment by AESilky** [comment]
The installation of the launcher should check for, and allow for installing, dependencies (PySerial, PySound, etc.).

**Comment by leskerr** [comment]
One of the things I've been most concerned about with a Python-based replacement for MorseKOB 2.5 is the hassle of installing all the necessary components and dependencies. Many MorseKOB users are uncomfortable with computers. Something like a Windows installer and its equivalent for other platforms would be wonderful.

**Comment by AESilky** [comment]
I agree - though I think we can defer this issue for a while. I wanted to create it because I think it will be important, but I think we should work on getting it running well and looking good on the three platforms first.

As we make progress there, we can work on this issue. I will make separate issues for 'Windows', 'Mac', 'Linux' and make them dependencies of this issue.  That way we can work on each platform in its own issue and then close this one when those are all closed.

**Comment by AESilky** [comment]
Depends on #48, #49 and #50

**Comment by AESilky** [comment]
Depends on #48, #49 and #50

**Comment by AESilky** [comment]
Created issue #73 for the installer mentioned in the description of this issue.

**Comment by AESilky** [comment]
Depends on #73
